obs-check-package-origin

```diff
@@ -0,0 +1,154 @@
+#!/bin/bash -e
+
+# Motivation: https://progress.opensuse.org/issues/154723
+# 1. fetch requirements from package spec file,
+# 2. look for package names for such requirements
+# 3. search all relevant instances where the package is maintained
+# 4. fetch version info from spec file and from Tumbleweed repo
+
+osc="${osc:-"osc --apiurl https://api.opensuse.org"}"
+
+BLUE="\e[94m"
+RED="\e[91m"
+GRAY="\e[90m"
+ENDCOLOR="\e[0m"
+
+debug() {
+    test -n "$DEBUG" && echo -e "${GRAY}$1${ENDCOLOR}" >&2
+}
+
+info() {
+    echo -e "${BLUE}$1${ENDCOLOR}" >&2
+}
+
+error() {
+    echo -e "${RED}$1${ENDCOLOR}" >&2
+}
+
+init_check() {
+    which yq &> /dev/null || {
```

Comment:
```suggestion
    command yq &> /dev/null || {
```

This was failing on a TW container until I changed it

Reply:
Hm, so `which` is not preinstalled in a TW container, good to know. Unfortunately `command` is not an equivalent of `which` here.

Reply:
> Hm, so `which` is not preinstalled in a TW container, good to know. Unfortunately `command` is not an equivalent of `which` here.

why so? but either way at least is known now

Reply:
If you are brave enough you can try the difference between `which vim` and `command vim`; but even if `command -v` is used, it behaves differently (the difference is in respect of bash functions and aliases, which will be found by command but would not work without interactive shell).